Lib/typing.py

```diff
@@ -2046,6 +2057,9 @@ def __class_getitem__(cls, params):
             raise TypeError("Annotated[...] should be used "
                             "with at least two arguments (a type and an "
                             "annotation).")
+        if _is_unpacked_typevartuple(params[0]):
```

Comment:
I don't think we need to explicitly disallow this at runtime.

Reply:
Hmm, but then how should we handle situations like this?

```python
A = Annotated[Unpack[Ts], "foo"]
B = A[int, str]
```

`Annotated[int, str, "foo"]` wouldn't be valid. We could do some kind of automatic tuple wrapping, `Annotated[tuple[int, str], "foo"`, but that would be inconsistent with how we handle `TypeVarTuple` substitution elsewhere.

Reply:
Good point, and sorry for not responding to this earlier.